src/settings/JobProfiles/JobProfiles.js

```diff
@@ -169,58 +161,32 @@ export class JobProfiles extends Component {
     const { location } = this.props;
     const { selectedRecords: { size: selectedRecordsSize } } = this.state;
 
-    return (
-      <Fragment>
-        <Button
-          data-test-new-job-profile-menu-button
-          to={createLayerURL(location, LAYER_TYPES.CREATE)}
-          buttonStyle="dropdownItem"
-          buttonClass={sharedCss.linkButton}
-          onClick={menu.onToggle}
-        >
-          <Icon icon="plus-sign">
-            <FormattedMessage id="ui-data-import.settings.jobProfiles.newJob" />
-          </Icon>
-        </Button>
-        <Button
-          data-test-export-selected-job-profiles-menu-button
-          buttonStyle="dropdownItem"
-          disabled={!selectedRecordsSize}
-          onClick={menu.onToggle}
-        >
-          <Icon icon="arrow-down">
-            <FormattedMessage id="ui-data-import.exportSelected" />
-            {!!selectedRecordsSize && (
-            <Fragment>
-              {' '}
-              <FormattedMessage
-                id="ui-data-import.itemsCount"
-                values={{ count: selectedRecordsSize }}
-              />
-            </Fragment>
-            )}
-          </Icon>
-        </Button>
-        <Button
-          data-test-select-all-job-profiles-menu-button
-          buttonStyle="dropdownItem"
-          onClick={() => this.handleSelectAllButton(menu)}
-        >
-          <Icon icon="select-all">
-            <FormattedMessage id="ui-data-import.selectAll" />
-          </Icon>
-        </Button>
-        <Button
-          data-test-deselect-all-job-profiles-menu-button
-          buttonStyle="dropdownItem"
-          onClick={() => this.handleDeselectAllButton(menu)}
-        >
-          <Icon icon="deselect-all">
-            <FormattedMessage id="ui-data-import.deselectAll" />
-          </Icon>
-        </Button>
-      </Fragment>
-    );
+    const config = {
+      items: [{
+        control: 'AddNew',
+        caption: 'ui-data-import.settings.jobProfiles.newProfile',
+        menu,
+        location,
+      }, {
+        control: 'ExportSelected',
+        menu,
+        selectedCount: selectedRecordsSize,
+      }, {
+        control: 'Default',
+        caption: 'ui-data-import.selectAll',
+        icon: 'check-circle',
+        onClick: () => this.handleSelectAllButton(menu),
+        dataAttributes: { 'data-test-select-all-items-menu-button': '' },
+      }, {
+        control: 'Default',
+        caption: 'ui-data-import.deselectAll',
+        icon: 'times-circle',
+        onClick: () => this.handleDeselectAllButton(menu),
+        dataAttributes: { 'data-test-deselect-all-items-menu-button': '' },
+      }],
+    };
+
+    return (<ActionMenu config={config} />);
```

Comment:
Write it the way bellow for consistency 

```
return (
  <ActionMenu config={config} />
);
```

Reply:
Better like that as it is one line
```suggestion
    return <ActionMenu config={config} />;
```